spec/swagger_helper.rb

```diff
@@ -30,7 +30,7 @@
 
           Read permissions are granted to any authorised client. No particular role is required.
 
-          Write permissions are granted to clients with the role `ROLE_COMPLEXITY_OF_NEED` and a `write` scope.
+          Write permissions are granted to clients with the role `ROLE_UPDATE_COMPLEXITY_OF_NEED`.
```

Comment:
This is incorrect – 

Read permissions require the role `ROLE_COMPLEXITY_OF_NEED`.